attributes/default.rb

```diff
@@ -23,6 +23,12 @@
 #
 default['chef-ingredient']['mixlib-install']['git_ref'] = nil
 
+#
+# Optionally override the omnitruck endpoint
+#
+default['chef-ingredient']['omnitruck']['endoint'] = nil
```

Comment:
Yes, I do believe that is the overall plan (to eliminate omnitruck altogether.)  There are several customizations that would have to be made to the Bintray backend to be able to wire it into mixlib-install as the default backend.

Serdar has made changes to omnitruck so that it now pulls from bintray.  This allows mixlib-install to use bintray proxied through omnitruck.  The reason I added this attribute is so I could test against the acceptance environment (since the omnitruck/bintray change has not been pushed to production).

This change will meet the requirement of downloading artifacts from packages.chef.io, and gives us time to make the necessary changes to mixlib-install's Bintray backend to wire it in as the default.

When this is complete, I can see removing references to omnitruck here.